src/dynamo/models/blog.ts

```diff
@@ -0,0 +1,257 @@
+import { DocumentClient } from 'aws-sdk2-types/lib/dynamodb/document_client';
+import { v4 as uuidv4 } from 'uuid';
+
+import BlogPost from '../../datatypes/BlogPost';
+import { BoardChanges, BoardType, Changes } from '../../datatypes/Card';
+import CubeType from '../../datatypes/Cube';
+import UserType from '../../datatypes/User';
+import { cardFromId } from '../../util/carddb';
+import createClient from '../util';
+import Changelog from './changelog';
+import Cube from './cube';
+import User from './user';
+
+enum FIELDS {
+  ID = 'id',
+  BODY = 'body',
+  OWNER = 'owner',
+  DATE = 'date',
+  CUBE_ID = 'cube',
+  TITLE = 'title',
+  CHANGELIST_ID = 'changelist',
+}
```

Comment:
I think we can refactor this out, since this was kind of a stopgap anyways. Instead we can just use typescript features accomplish this

Reply:
I like `UnhydratedBlog`, and `Blog`, yeah. These enums are really just a stopgap and not are are just clutter now

Reply:
@dekkerglen I've pushed a new commit in which I added UnhydratedBlogPost and make BlogPost "extend" (as best possible in typescript) it, removing the FIELDS array fully.

One annoyance I found was the lack of way to align the types with the dynamo table configuration, property names like "cube" have to be duplicated as strings in the config. Which isn't a huge deal as unless working directly in the model you don't really need to know.

While making the updates I did notice that the body of a Blog post could be null, which normally is only when its an automated post that has a changelog. However I did see that the create blog post server function does not require body to be a string with length > 0, though it does check that for title. Should I add that in?